### PR TITLE
Add riegeli 0.0.0-20241218-3385e3c

### PR DESCRIPTION
**Pull request by dryman**
This version is required by ArrayRecord

Last PR attempt didn't work because I needed to add `@bazel-io skip_check unstable_url` in the PR comment.

### CODE REVIEW

**Comment by dryman** [comment]
@bazel-io skip_check unstable_url

**Comment by bazel-io** [comment]
Hello @QrczakMK, modules you maintain (riegeli) have been updated in this PR. Please review the changes.

File: modules/riegeli/0.0.0-20241218-3385e3c/presubmit.yml
Comment (fmeum) [inline modules/riegeli/0.0.0-20241218-3385e3c/presubmit.yml:8]:
```suggestion
  - 7.x
  - 8.x
```

**Review by fmeum — APPROVED** [review]

**Review by QrczakMK — APPROVED** [review]

**Review by bazel-io — APPROVED** [review]
Hello @bazelbuild/bcr-maintainers, all modules in this PR have been approved by their maintainers. Please take a final look to merge this PR.

**Comment by dryman** [comment]
@QrczakMK we need your approval again because I made a change to lock bazel to 7.x. (one of the upstream package lzma/xz doesn't build with 8.x)

**Review by QrczakMK — APPROVED** [review]
It would be great to fix lzma so that it builds with bazel 8.x.

**Review by bazel-io — APPROVED** [review]
Hello @bazelbuild/bcr-maintainers, all modules in this PR have been approved by their maintainers. Please take a final look to merge this PR.

**Pull request merged**
